### PR TITLE
[Selenium] Update java selenium tests

### PR DESCRIPTION
**Pull request by SkorikSergey**
### What does this PR do?
[Selenium] Update java selenium tests according to removing 'New Workspace' page

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/16899

### CODE REVIEW

**Comment by SkorikSergey** [comment]
[ci-test]

**Comment by SkorikSergey** [comment]
crw-ci-test

**Comment by SkorikSergey** [comment]
crw-ci-test

**Comment by SkorikSergey** [comment]
[ci-test]

**Comment by SkorikSergey** [comment]
[ci-test]

**Comment by SkorikSergey** [comment]
[ci-test]

**Comment by SkorikSergey** [comment]
[ci-test]

**Comment by SkorikSergey** [comment]
[ci-test]

**Comment by SkorikSergey** [comment]
[ci-test]

**Comment by SkorikSergey** [comment]
[ci-test]

**Comment by SkorikSergey** [comment]
[ci-test]

**Comment by SkorikSergey** [comment]
[ci-test]

**Comment by SkorikSergey** [comment]
[ci-test]

**Comment by SkorikSergey** [comment]
[ci-test]

**Comment by SkorikSergey** [comment]
[ci-test]

**Comment by SkorikSergey** [comment]
[ci-test]

**Comment by SkorikSergey** [comment]
[ci-test]

**Comment by SkorikSergey** [comment]
crw-ci-test

File: tests/legacy-e2e/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/AddWorkspaceToOrganizationTest.java [hunk 1]
```diff
@@ -33,7 +33,7 @@
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
-@Test(groups = {TestGroup.MULTIUSER, TestGroup.OPENSHIFT, TestGroup.K8S})
+@Test(groups = {UNDER_REPAIR})
```
Comment (dmytro-ndp) [inline tests/legacy-e2e/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/AddWorkspaceToOrganizationTest.java]: You can just add `TestGroup.UNDER_REPAIR` item to the list not removing existed ones.

File: tests/legacy-e2e/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/ShareWorkspaceMemberTest.java [hunk 2]
```diff
@@ -43,16 +41,16 @@
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
-@Test(groups = {TestGroup.MULTIUSER, TestGroup.OPENSHIFT, TestGroup.K8S, TestGroup.UNDER_REPAIR})
+@Test(groups = {UNDER_REPAIR})
```
Comment (dmytro-ndp) [inline tests/legacy-e2e/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/ShareWorkspaceMemberTest.java]: You can just add `TestGroup.UNDER_REPAIR` item to the list not removing existed ones.

**Comment by dmytro-ndp** [comment]
Let's remove organization e2e tests files as well, because it's hardly ever existed organization UI i dashboard will be restored in future.

**Comment by dmytro-ndp** [comment]
There are two more tests removed compare to the list of tests to remove defined in https://github.com/eclipse/che/issues/16899:
- RenameWorkspaceTest
- RecreateUpdateStrategyWithStartedWorkspaceTest

I am curious if we can't leave them, because, you know, it would be better to have as highest test coverage as possible.

**Comment by SkorikSergey** [comment]
[ci-test]

**Comment by SkorikSergey** [comment]
[ci-test]

**Comment by SkorikSergey** [comment]
crw-ci-test

**Review by dmytro-ndp — APPROVED** [review]

**Comment by SkorikSergey** [comment]
[ci-test]

**Pull request merged**